jenkins/Jenkinsfile

```diff
@@ -7,7 +7,7 @@ pipeline {
   stages {
     stage('GPU Testing') {
       parallel {
-        stage('NDTensorsCUDAExt julia-1.6') {
+        stage('NDTensorsCUDAExt julia-1.8') {
```

Comment:
Reminder to delete this section of the workflow to not test against Julia 1.8.

Reply:
This section has been deleted. Thanks!